libstuff/SQResult.cpp

```diff
@@ -183,3 +188,55 @@ SQResult& SQResult::operator=(const SQResult& other) {
     }
     return *this;
 }
+
+vector<SQResultRow>::iterator SQResult::begin() {
+    return rows.begin();
+}
+
+vector<SQResultRow>::iterator SQResult::end() {
+    return rows.end();
+}
```

Comment:
Do we need the two non-const methods? I wonder if we can get away without having them and thus exposing our internals to be modified. Can you check if these are used anywhere outside of `MySQL.cpp`? If they're only used there, maybe we can remove them and come up with a workaround.